Migrate internal iOS 18 test baselines/expectations to OpenSource.

bls1999 opened the pull request:
#### 4840a7f165cbfa5794f7196737072336469ebb1c
<pre>
Migrate internal iOS 18 test baselines/expectations to OpenSource.
<a href="https://bugs.webkit.org/show_bug.cgi?id=276370">https://bugs.webkit.org/show_bug.cgi?id=276370</a>
<a href="https://rdar.apple.com/131389611">rdar://131389611</a>

Reviewed by Ryan Haddad.

Migrating iOS 18 layout test baselines and expectations from Internal -&gt; OpenSource.

* LayoutTests/platform/ios-17/TestExpectations: Added.
* LayoutTests/platform/ios-17/css2.1/t0805-c5518-brdr-t-01-e-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0805-c5518-brdr-t-01-e-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0805-c5519-brdr-r-00-a-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0805-c5519-brdr-r-00-a-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0805-c5520-brdr-b-01-e-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0805-c5520-brdr-b-01-e-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0805-c5521-brdr-l-00-a-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0805-c5521-brdr-l-00-a-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0805-c5521-brdr-l-01-e-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0805-c5521-brdr-l-01-e-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0805-c5521-ibrdr-l-00-a-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0805-c5521-ibrdr-l-00-a-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c414-flt-02-c-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c414-flt-02-c-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c414-flt-03-c-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c414-flt-03-c-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c414-flt-04-c-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c414-flt-04-c-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c5525-fltblck-01-d-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c5525-fltblck-01-d-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c5525-fltcont-00-d-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c5525-fltcont-00-d-g-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c5525-fltwidth-00-c-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c5525-fltwidth-00-c-g-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c5525-fltwidth-02-c-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c5525-fltwidth-02-c-g-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c5525-fltwidth-03-c-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c5525-fltwidth-03-c-g-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c5525-fltwrap-00-b-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c5525-fltwrap-00-b-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t090501-c414-flt-01-b-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t090501-c414-flt-01-b-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t090501-c414-flt-03-b-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t090501-c414-flt-03-b-g-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t090501-c5525-flt-l-00-b-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t090501-c5525-flt-l-00-b-g-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t090501-c5525-flt-r-00-b-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t090501-c5525-flt-r-00-b-g-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t100304-c43-rpl-bbx-01-d-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t100304-c43-rpl-bbx-01-d-g-expected.txt.
* LayoutTests/platform/ios-17/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-in-loose-expected.txt: Copied from LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-in-loose-expected.txt.
* LayoutTests/platform/ios-17/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-pr-normal-expected.txt: Copied from LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-pr-normal-expected.txt.
* LayoutTests/platform/ios-17/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-pr-strict-expected.txt: Copied from LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-pr-strict-expected.txt.
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/ios/css2.1/t0805-c5518-brdr-t-01-e-expected.txt:
* LayoutTests/platform/ios/css2.1/t0805-c5519-brdr-r-00-a-expected.txt:
* LayoutTests/platform/ios/css2.1/t0805-c5520-brdr-b-01-e-expected.txt:
* LayoutTests/platform/ios/css2.1/t0805-c5521-brdr-l-00-a-expected.txt:
* LayoutTests/platform/ios/css2.1/t0805-c5521-brdr-l-01-e-expected.txt:
* LayoutTests/platform/ios/css2.1/t0805-c5521-ibrdr-l-00-a-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c414-flt-02-c-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c414-flt-03-c-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c414-flt-04-c-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c5525-fltblck-01-d-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c5525-fltcont-00-d-g-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c5525-fltwidth-00-c-g-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c5525-fltwidth-02-c-g-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c5525-fltwidth-03-c-g-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c5525-fltwrap-00-b-expected.txt:
* LayoutTests/platform/ios/css2.1/t090501-c414-flt-01-b-expected.txt:
* LayoutTests/platform/ios/css2.1/t090501-c414-flt-03-b-g-expected.txt:
* LayoutTests/platform/ios/css2.1/t090501-c5525-flt-l-00-b-g-expected.txt:
* LayoutTests/platform/ios/css2.1/t090501-c5525-flt-r-00-b-g-expected.txt:
* LayoutTests/platform/ios/css2.1/t100304-c43-rpl-bbx-01-d-g-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-in-loose-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-pr-normal-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-pr-strict-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/zh/css-text-line-break-zh-pr-normal-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/zh/css-text-line-break-zh-pr-strict-expected.txt: Added.

Canonical link: <a href="https://commits.webkit.org/280829@main">https://commits.webkit.org/280829@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/c2b9369b127e69b3ccdcd113caecae9d42711c8c

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/57757 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/37085 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/10233 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/61379 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/8202 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/44721 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/8390 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/46797 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/5817 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/59787 "Passed tests") | [❌ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/34780 "Exiting early after 10 failures. 30 tests run. 1 failures") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/49907 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/27622 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/31550 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/7200 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/7206 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/53502 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/7470 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/63062 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/1671 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/62/builds/7547 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/54014 "Found 2 new test failures: imported/w3c/web-platform-tests/html/semantics/embedded-content/the-iframe-element/sandbox-inherit-to-blank-document-unsandboxed-frame.html webgl/2.0.0/conformance/canvas/rapid-resizing.html (failure)") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/1677 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/49918 "Passed tests") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/54132 "Found 2 new API test failures: /WebKitGTK/TestWebKitAccessibility:/webkit/WebKitAccessibility/text/state-changed, /WebKitGTK/TestInspectorServer:/webkit/WebKitWebInspectorServer/http-test-page-list (failure)") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/1416 "Passed tests") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/8610 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/32914 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/34000 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/35084 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/33745 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

ryanhaddad reviewed (approved):
rs=me if iOS EWS is happy.

bls1999 committed:
Migrate internal iOS 18 test baselines/expectations to OpenSource.
https://bugs.webkit.org/show_bug.cgi?id=276370
rdar://131389611

Reviewed by Ryan Haddad.

Migrating iOS 18 layout test baselines and expectations from Internal -> OpenSource.

* LayoutTests/platform/ios-17/TestExpectations: Added.
* LayoutTests/platform/ios-17/css2.1/t0805-c5518-brdr-t-01-e-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0805-c5518-brdr-t-01-e-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0805-c5519-brdr-r-00-a-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0805-c5519-brdr-r-00-a-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0805-c5520-brdr-b-01-e-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0805-c5520-brdr-b-01-e-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0805-c5521-brdr-l-00-a-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0805-c5521-brdr-l-00-a-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0805-c5521-brdr-l-01-e-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0805-c5521-brdr-l-01-e-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0805-c5521-ibrdr-l-00-a-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0805-c5521-ibrdr-l-00-a-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c414-flt-02-c-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c414-flt-02-c-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c414-flt-03-c-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c414-flt-03-c-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c414-flt-04-c-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c414-flt-04-c-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c5525-fltblck-01-d-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c5525-fltblck-01-d-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c5525-fltcont-00-d-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c5525-fltcont-00-d-g-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c5525-fltwidth-00-c-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c5525-fltwidth-00-c-g-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c5525-fltwidth-02-c-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c5525-fltwidth-02-c-g-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c5525-fltwidth-03-c-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c5525-fltwidth-03-c-g-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t0905-c5525-fltwrap-00-b-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t0905-c5525-fltwrap-00-b-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t090501-c414-flt-01-b-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t090501-c414-flt-01-b-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t090501-c414-flt-03-b-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t090501-c414-flt-03-b-g-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t090501-c5525-flt-l-00-b-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t090501-c5525-flt-l-00-b-g-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t090501-c5525-flt-r-00-b-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t090501-c5525-flt-r-00-b-g-expected.txt.
* LayoutTests/platform/ios-17/css2.1/t100304-c43-rpl-bbx-01-d-g-expected.txt: Copied from LayoutTests/platform/ios/css2.1/t100304-c43-rpl-bbx-01-d-g-expected.txt.
* LayoutTests/platform/ios-17/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-in-loose-expected.txt: Copied from LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-in-loose-expected.txt.
* LayoutTests/platform/ios-17/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-pr-normal-expected.txt: Copied from LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-pr-normal-expected.txt.
* LayoutTests/platform/ios-17/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-pr-strict-expected.txt: Copied from LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-pr-strict-expected.txt.
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/ios/css2.1/t0805-c5518-brdr-t-01-e-expected.txt:
* LayoutTests/platform/ios/css2.1/t0805-c5519-brdr-r-00-a-expected.txt:
* LayoutTests/platform/ios/css2.1/t0805-c5520-brdr-b-01-e-expected.txt:
* LayoutTests/platform/ios/css2.1/t0805-c5521-brdr-l-00-a-expected.txt:
* LayoutTests/platform/ios/css2.1/t0805-c5521-brdr-l-01-e-expected.txt:
* LayoutTests/platform/ios/css2.1/t0805-c5521-ibrdr-l-00-a-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c414-flt-02-c-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c414-flt-03-c-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c414-flt-04-c-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c5525-fltblck-01-d-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c5525-fltcont-00-d-g-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c5525-fltwidth-00-c-g-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c5525-fltwidth-02-c-g-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c5525-fltwidth-03-c-g-expected.txt:
* LayoutTests/platform/ios/css2.1/t0905-c5525-fltwrap-00-b-expected.txt:
* LayoutTests/platform/ios/css2.1/t090501-c414-flt-01-b-expected.txt:
* LayoutTests/platform/ios/css2.1/t090501-c414-flt-03-b-g-expected.txt:
* LayoutTests/platform/ios/css2.1/t090501-c5525-flt-l-00-b-g-expected.txt:
* LayoutTests/platform/ios/css2.1/t090501-c5525-flt-r-00-b-g-expected.txt:
* LayoutTests/platform/ios/css2.1/t100304-c43-rpl-bbx-01-d-g-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-in-loose-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-pr-normal-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/ja/css-text-line-break-ja-pr-strict-expected.txt:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/zh/css-text-line-break-zh-pr-normal-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/css/css-text/i18n/zh/css-text-line-break-zh-pr-strict-expected.txt: Added.

Canonical link: https://commits.webkit.org/280829@main
